python-backend/app/api/incidents/response_models.py

```diff
@@ -1,6 +1,12 @@
 from app.extensions import api
 from flask_restplus import fields
 
+
+class DateTime(fields.Raw):
+    def format(self, value):
+        return value.strftime("%Y-%m-%d %H:%M") if value else None
```

Comment:
What's this for? Are we formatting timestamps like this elsewhere?

If we are, we need to be very sure that we accept data in the same format that we return it.